css-two-column-layout/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column-full">
+        <h1>SCALE, SPEED, AGILITY</h1>
+        <p>We've dedicated thousands of man hours to making healthcare institutions in the Helium network run 10x more efficiently and deliver an unparalleled quality of care</p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="images">
+        <img src="images/software.svg" alt="image of software that looks like Africa">
+      </div>
+      <div class="two-fifths">
+        <h3>INTUITIVE,<br>SMART SOFTWARE</h3>
+        <p>We build products that transform how healthcare facilties run across Africa &#8212; making them radically more efficient operationally and financially, a complete turnaround.</p>
+      </div>
```

Comment:
the only thing to add would be to wrap these elements with a `column-half` element which will be the empty box for half of the row.